tools/src/main/java/org/apache/kafka/tools/ClientMetricsCommand.java

```diff
@@ -241,7 +241,7 @@ public ClientMetricsCommandOptions(String[] args) {
             intervalOpt = parser.accepts("interval", "The metrics push interval in milliseconds.")
```

Comment:
Could you add this "reset the interval" usage into document?

Reply:
we should do that for other configs as well.